### PR TITLE
docs: tip for legacy browser

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |      -     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: .dumi/theme/slots/Footer/AdditionalInfo.tsx [hunk 1]
```diff
@@ -1,194 +1,89 @@
...
-                    .
-                  </p>
-                </>
-              )}
-            </Typography>
-          </Col>
-        </Row>
-      </Modal> */}
-    </>
+        {location.whereNotSupport}{' '}
```
Comment (yoyo837) [inline .dumi/theme/slots/Footer/AdditionalInfo.tsx:82]:
```suggestion
        {location.whereNotSupport}
```

Comment (zombieJ) [reply .dumi/theme/slots/Footer/AdditionalInfo.tsx:82]: 这个故意的，否则英文版会贴的比较紧哈~

**Comment by zombieJ** [comment]
#### 360
截屏2022-11-24 00 05 19

**Review by afc163 — APPROVED** [review]

File: .dumi/theme/slots/Footer/AdditionalInfo.tsx [hunk 2]
```diff
@@ -1,194 +1,89 @@
...
+    whereDocTitle: '兼容性调整（请使用现代浏览器访问）',
+    whereDocUrl: '/docs/react/customize-theme-cn#兼容性调整',
   },
   en: {
-    title: '🎉🎉🎉 Ant Design 5.0 is released! 🎉🎉🎉',
-    ok: 'Got it',
+    whereNotSupport:
+      'Your browser not support modern CSS Selector. Please use modern browser to view. If you want to compatible style with legacy browser, please refer to the configuration document:',
+    whereDocTitle: 'Compatible adjustment (Please use modern browser to visit)',
+    whereDocUrl: '/docs/react/customize-theme#compatible-adjustment',
```
Comment (afc163) [inline .dumi/theme/slots/Footer/AdditionalInfo.tsx:18]: 这个链接估计用户也看不了，不如跳一个 u.ant.design 短链再跳 issue。

Comment (zombieJ) [reply .dumi/theme/slots/Footer/AdditionalInfo.tsx:18]: 文案里提示用户用 现代浏览器 打开链接，跳太多其实它最终还是得用现代浏览器。就不绕圈圈了。

Comment (afc163) [reply .dumi/theme/slots/Footer/AdditionalInfo.tsx:18]: 用户可能需要先去链接里看看哪些是现代浏览器。

File: .dumi/theme/slots/Footer/AdditionalInfo.tsx [hunk 3]
```diff
@@ -1,194 +1,89 @@
...
-import useSiteToken from '../../../hooks/useSiteToken';
+
+const whereCls = 'ant-where-checker';
 
 const locales = {
   cn: {
-    title: '🎉🎉🎉 Ant Design 5.0 发布！ 🎉🎉🎉',
-    ok: '知道了',
+    whereNotSupport:
+      '你的浏览器不支持现代 CSS Selector，请使用现代浏览器查看官网。如果需要对旧版浏览器进行样式支持，欢迎查阅配置文档：',
```
Comment (afc163) [inline .dumi/theme/slots/Footer/AdditionalInfo.tsx:10]:
```suggestion
      '你的浏览器 ${navigator.userAgent} 不支持现代 CSS Selector，请使用现代浏览器查看官网。如果需要对旧版浏览器进行样式支持，欢迎查阅配置文档：',
```

**Comment by zombieJ** [comment]
> Modal 弹层提示是不是就行了

前面没看到这个，Modal 弹层一样会因为不支持 `:where` 而样式飞掉

File: .dumi/theme/slots/Footer/AdditionalInfo.tsx [hunk 4]
```diff
@@ -1,194 +1,89 @@
...
-import useSiteToken from '../../../hooks/useSiteToken';
+
+const whereCls = 'ant-where-checker';
 
 const locales = {
   cn: {
-    title: '🎉🎉🎉 Ant Design 5.0 发布！ 🎉🎉🎉',
-    ok: '知道了',
+    whereNotSupport:
+      '你的浏览器 ${navigator.userAgent} 不支持现代 CSS Selector，请使用现代浏览器查看官网。如果需要对旧版浏览器进行样式支持，欢迎查阅配置文档：',
```
Comment (afc163) [inline .dumi/theme/slots/Footer/AdditionalInfo.tsx:10]:
```suggestion
      `你的浏览器 ${navigator.userAgent} 不支持现代 CSS Selector，请使用现代浏览器查看官网。如果需要对旧版浏览器进行样式支持，欢迎查阅配置文档：`,
```

**Pull request merged**